clients/python/test/e2e/test_files_api.py

```diff
@@ -33,7 +35,30 @@ def _hash_file(file: Path) -> str:
 def test_upload_file(
     create_tmp_file: Callable[[ByteSize], Path], api_client: osparc.ApiClient
 ) -> None:
-    """Test that we can upload a file via the multipart upload and download it again. Also check RAM usage of upload/download fcns"""
+    """Test that we can upload a file via the multipart upload and download it again."""
+    tmp_file = create_tmp_file(ByteSize(1 * _GB))
+    tmp_path: Path = tmp_file.parent
+    files_api: osparc.FilesApi = osparc.FilesApi(api_client=api_client)
+    try:
+        uploaded_file1: osparc.File = files_api.upload_file(tmp_file)
+        uploaded_file2: osparc.File = files_api.upload_file(tmp_file)
+        assert (
+            uploaded_file1.id == uploaded_file2.id
+        ), "could not detect that file was already on server"
+        downloaded_file = files_api.download_file(
+            uploaded_file1.id, destination_folder=tmp_path, retval=True
+        )
+        assert Path(downloaded_file).parent == tmp_path
+        assert _hash_file(Path(downloaded_file)) == _hash_file(tmp_file)
+    finally:
+        files_api.delete_file(uploaded_file1.id)
```

Comment:
THOUGHT: perhpas we should have uploads with expiration time ... and have them enough time for the duration of the test

Reply:
Yes, I think we should have a default expiration date for all uploaded files. Because I doubt people are very diligent about deleting their files on the server.  Dustin also pointed this out some time ago.